index.html

```diff
@@ -0,0 +1,591 @@
+            <div class="social-share">
+              <span>
+                <img
+                  src="https://cdn3.iconfinder.com/data/icons/social-media-black-white-2/512/BW_Facebook_2_glyph_svg-128.png"
+                  srcset="
+                    https://cdn3.iconfinder.com/data/icons/social-media-black-white-2/512/BW_Facebook_2_glyph_svg-128.png 1x,
+                    https://cdn3.iconfinder.com/data/icons/social-media-black-white-2/512/BW_Facebook_2_glyph_svg-256.png 2x
+                  "
+                  alt=" icon"
+                  loading="lazy"
+                />
+              </span>
+              <span>
+                <img
+                  src="https://cdn3.iconfinder.com/data/icons/social-media-black-white-2/512/BW_Whatsapp_glyph_svg-128.png"
+                  srcset="
+                    https://cdn3.iconfinder.com/data/icons/social-media-black-white-2/512/BW_Whatsapp_glyph_svg-128.png 1x,
+                    https://cdn3.iconfinder.com/data/icons/social-media-black-white-2/512/BW_Whatsapp_glyph_svg-256.png 2x
+                  "
+                  alt=" icon"
+                  class="d-block"
+                  loading="lazy"
+                />
+              </span>
+              <span>
+                <img
+                  src="https://cdn3.iconfinder.com/data/icons/social-media-black-white-2/512/BW_Twitter_glyph_svg-128.png"
+                  srcset="
+                    https://cdn3.iconfinder.com/data/icons/social-media-black-white-2/512/BW_Twitter_glyph_svg-128.png 1x,
+                    https://cdn3.iconfinder.com/data/icons/social-media-black-white-2/512/BW_Twitter_glyph_svg-256.png 2x
+                  "
+                  alt=" icon"
+                  class="d-block"
+                  loading="lazy"
+                />
+              </span>
+              <span>
+                <img
+                  src="https://cdn4.iconfinder.com/data/icons/ionicons/512/icon-email-128.png"
+                  srcset="
+                    https://cdn4.iconfinder.com/data/icons/ionicons/512/icon-email-256.png 2x
+                  "
+                  alt="email-icon"
+                />
+              </span>
+              <span>
+                <img
+                  src="https://cdn4.iconfinder.com/data/icons/icon-flat-icon-set/50/share-raw-128.png"
+                  srcset="
+                    https://cdn4.iconfinder.com/data/icons/icon-flat-icon-set/50/share-raw-256.png 2x
+                  "
+                  alt="share, arrow icon"
+                />
+              </span>
+            </div>
+          </div>
+        </div>
+      </div>
+    </main>
+
+    <footer>
+      <hr/>
+      <hr/>
+
+      <div class="footer-header flex">
+        <div>
+          <img
+            src="https://raw.githubusercontent.com/mrlepak/NYT-Replica/main/assets/Logo.png"
+            alt="https://raw.githubusercontent.com/mrlepak/NYT-Replica/main/assets/Logo.png"
```

Comment:
[optional] Make sure that you use a descriptive texts on the alt attribute, of an image because it helps on search engine optimization